src/context-selection/attribute-option-combo-selector-bar-item/index.js

```diff
@@ -1,2 +1,2 @@
 export { default as AttributeOptionComboSelectorBarItem } from './attribute-option-combo-selector-bar-item.js'
-export { useAttributeParams } from './use-attribute-params.js'
+export { useApiAttributeParams } from '../../data-workspace/use-api-attribute-params.js'
```

Comment:
This code artifact can be deleted, I suppose VS Code did this "magically" when moving the file